test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -49,9 +49,32 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
     assert_equal image.image_data.file_url, images.first['url']
   end
 
+  test "appends detailed document type information about an edition" do
+    news = create(:news_article)
+    publication = create(:publication)
+    speech = create(:speech)
+    corpinfo = create(:corporate_information_page)
+
+    newsresult = DocumentExportPresenter.new(news.document).as_json
+    doctype_data = newsresult[:editions].first[:document_type_information]
+    assert_equal news.news_article_type.as_json, doctype_data
+
+    pubresult = DocumentExportPresenter.new(publication.document).as_json
+    doctype_data = pubresult[:editions].first[:document_type_information]
+    assert_equal publication.publication_type.as_json, doctype_data
+
+    speechresult = DocumentExportPresenter.new(speech.document).as_json
+    doctype_data = speechresult[:editions].first[:document_type_information]
+    assert_equal speech.speech_type.as_json, doctype_data
+
+    corpresult = DocumentExportPresenter.new(corpinfo.document).as_json
+    doctype_data = corpresult[:editions].first[:document_type_information]
+    assert_equal corpinfo.corporate_information_page_type.as_json, doctype_data
```

Comment:
You seem to have stopped separating words with underscores here?

Reply:
Tired error.